### PR TITLE
Xử lý Authentication

### PR DESCRIPTION
**Pull request by methanol-dev**
## Related Tickets:
- [#47098](https://edu-redmine.sun-asterisk.vn/issues/47098)

## WHAT this PR does?
- Xử lý Authentication

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by methanol-dev** [comment]
ready

File: resources/views/layouts/app.blade.php [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                                </a>
+
+                                <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown">
+                                    <a class="dropdown-item" href="{{ route('logout') }}"
+                                       onclick="event.preventDefault();
+                                                     document.getElementById('logout-form').submit();">
+                                        {{ __('Logout') }}
+                                    </a>
+
+                                    <form id="logout-form" action="{{ route('logout') }}" method="POST" style="display: none;">
```
Comment (ghost) [inline resources/views/layouts/app.blade.php]: bỏ style ra file css riêng

File: resources/views/layouts/app.blade.php [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+                        @else
+                            <li class="nav-item dropdown">
+                                <a id="navbarDropdown" class="nav-link dropdown-toggle" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" v-pre>
+                                    {{ Auth::user()->name }} <span class="caret"></span>
+                                </a>
+
+                                <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown">
+                                    <a class="dropdown-item" href="{{ route('logout') }}"
+                                       onclick="event.preventDefault();
+                                                     document.getElementById('logout-form').submit();">
```
Comment (ghost) [inline resources/views/layouts/app.blade.php]: bỏ onclick event ra file js riêng

**Comment by methanol-dev** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**